common/src/web/script_page.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
We don't require a new html page for just setting `window.foo`, you could do it inline in the tests - `data:text/html,<script>window.foo=123;</script>`